tests/integration_tests/test_examples.py

```diff
@@ -238,6 +239,18 @@ def test_rl2_ppo_halfcheetah():
                           check=False).returncode == 0
 
 
+@pytest.mark.mujoco
```

Comment:
why does this need to be in the manual test cases rather than automatically-collected above?

@gitanshu PTAL

Reply:
I wanted to give it smaller parameters to save time. It's ok to just leave it automatically-collected as well.

Reply:
i think that n_epochs should always be 1 in tests?

I'm fine with it as long as @gitanshu is.

Reply:
Yeah, automatically-collected examples are run so that the runner only runs one epoch, so I don't think this should really be necessary.